src/components/MessageBox.js

```diff
@@ -115,7 +122,7 @@ class MessageBox extends React.Component {
       messageContent: {
         boxSizing: this.props.isSmall ? null : 'content-box',
         textAlign: 'left',
-        width: this.props.isSmall ? '100%' : '75%',
+        width: this.props.children ? '100%' : '75%',
```

Comment:
Was it intentional to remove the `isSmall` prop from changing the width?

Reply:
For backwards compatibility we should leave it. I have changed it to an or. When it small it should take 100%, but also when there are children it should be 100%.